src/components/feedbackFaces.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { event } from "@site/src/utils/gtags.client";
+
+// Default export function component
+export default function FeedbackFaces() {
+    const [clickedFace, setClickedFace] = useState<string | null>(null);
+
+    const faces = [
+        { label: "Sad", emoji: "😞" },
+        { label: "Neutral", emoji: "😐" },
+        { label: "Happy", emoji: "😊" },
+    ];
+
+    const handleFeedbackClick = (feedbackType: string) => {
+        setClickedFace(feedbackType);
+
+        const pagePath = window.location.pathname;
+        const label = `${feedbackType} - ${pagePath}`;
+
+        event({
+            action: "feedback_click",
+            category: "Feedback",
+            label: label,
```

Comment:
Would also probably lowercase here, but just a nit. Do you also want to use the user hash so that you can tell how many unique people are clicking it and reduce the number of multiple clicks?

Reply:
I actually want multiple clicks here.  Sort of like clapping on Medium vs. upvotes or downvotes.